src/agent/debuglet.ts

```diff
@@ -14,38 +14,32 @@
  * limitations under the License.
  */
 
-import {AuthenticationConfig, Common} from '../types/common';
-const common: Common = require('@google-cloud/common');
-
+import * as assert from 'assert';
```

Comment:
Why import assert?

Reply:
We have always imported `assert` – there are some debugging assertions in the code. `gts fix` reorganized the imports for whatever reason.